### PR TITLE
Another dialog

### PR DESCRIPTION
**Pull request by orangejenny**
another https://github.com/dimagi/Vellum/pull/433

@proteusvacuum

### CODE REVIEW

File: src/core.js [hunk 1]
```diff
@@ -696,17 +696,8 @@ define([
...
-            })),
-            $body = $("body"),
-            keyup = function (event) {
-                if (event.keyCode === 27) { // escape
-                    $modal.modal('hide');
-                }
-            };
-        $body.on("keyup", keyup);
-        $modal.on('hide', function () {
-            $body.off("keyup", keyup);
```
Comment (millerdev) [inline src/core.js]: Why did you remove this keyup code? This makes the escape key dismiss the dialog.

Comment (orangejenny) [inline src/core.js]: That's how bootstrap modals behave by default (the `keyboard` parameter defaults to true) and having this extra handler prevented me from turning off that behavior.

Comment (millerdev) [inline src/core.js]: > That's how bootstrap modals behave by default...

Oh, cool. Didn't realize that.
